Core/LocalAdmin.cs

```diff
@@ -672,14 +726,68 @@ private static void SetupKeyboardInput()
     {
         new Task(() =>
         {
+            ConsoleKeyInfo consoleKeyInfo;
+            int historyIndex = -1;
+
+            static void AddToCommandHistory(string command)
+            {
+                if (_commandsHistory.Contains(command))
+                {
+                    _commandsHistory.Remove(command);
+                }
+                else if (_commandsHistory.Count == CommandsHistorySize)
+                {
+                    _commandsHistory.RemoveAt(_commandsHistory.Count - 1);
+                }
```

Comment:
This can be crushed for better readability.

```suggestion
                if (_commandsHistory.Contains(command))
                    _commandsHistory.Remove(command);
                else if (_commandsHistory.Count == CommandsHistorySize)
                    _commandsHistory.RemoveAt(_commandsHistory.Count - 1);
```

Reply:
Then you can just hit commit to add this

Reply:
Why do you use Contains check? It's completely redundant as Remove already checks that for you and returns a bool.

Just take a look at the docs: https://learn.microsoft.com/en-us/dotnet/api/system.collections.generic.list-1.remove?view=net-8.0

and how List.Remove() is implemented in .NET 8:

```cs
        // Removes the first occurrence of the given element, if found.
        // The size of the list is decreased by one if successful.
        public bool Remove(T item)
        {
            int index = IndexOf(item);
            if (index >= 0)
            {
                RemoveAt(index);
                return true;
            }

            return false;
        }
```